### PR TITLE
Add ART library to UIExplorer for iOS, with sample and snapshot test

### PR DESCRIPTION
**Pull request by dlowder-salesforce**
**Motivation**

The ART library is part of the react-native repo, but is not included in UIExplorer and has no native testing.  This PR adds the ART library to UIExplorer, adds an example tab for it, and adds a snapshot test.

**Test plan**

New snapshot test.

### CODE REVIEW

File: Examples/UIExplorer/js/ARTExample.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+        return(
+            <View>
+                <Surface width={scale*200} height={scale*200}>
+                    <Group>
+                        <Shape d={pathRect} stroke="#000080" fill="#000080" strokeWidth={scale*1}/>
+                        <Shape d={pathCircle0} stroke="#FF0000" fill="#FF0000" strokeWidth={scale*1}/>
+                        <Shape d={pathCircle1} stroke="#00FF00" fill="#00FF00" strokeWidth={scale*1}/>
+                        <Shape d={pathCircle2} stroke="#00FFFF" fill="#00FFFF" strokeWidth={scale*1}/>
+                        <Shape d={pathCircle3} stroke="#FFFFFF" fill="#FFFFFF" strokeWidth={scale*1}/>
```
Comment (browniefed) [inline Examples/UIExplorer/js/ARTExample.js]: `*1`?

**Comment by browniefed** [comment]
This example may be confusing. Maybe just use the https://github.com/reactjs/react-art/tree/master/examples/vector-widget example here?

File: Examples/UIExplorer/js/ARTExample.js (hunk 1 above)
Comment (dlowder-salesforce) [inline Examples/UIExplorer/js/ARTExample.js]: LOL that's what happens when you do search and replace text too quickly....

**Comment by dlowder-salesforce** [comment]
@browniefed that's an animation, not a still view.... I wanted something static to make snapshot testing reliable, and also want it to scale in size for both tvOS and iOS views.

**Comment by hramos** [comment]
Thanks for adding this!

**Pull request closed without merging**